### PR TITLE
docs: Add document for v5 direction to share with the broader community

### PR DESCRIPTION
**Pull request by bryantbiggs**
### What does this PR do?

🛑 Please open an issue first to discuss any significant work and flesh out details/direction - we would hate for your time to be wasted.
Consult the [CONTRIBUTING](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/CONTRIBUTING.md#contributing-via-pull-requests) guide for submitting pull-requests.

### Motivation

- Resolves #<issue-number>

### More

- [ ] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [ ] Yes, I have added a new example under [examples](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR for add-ons under [add-ons](https://github.com/aws-samples/eks-blueprints-add-ons) repo (if applicable)
- [ ] Yes, I have updated the [docs](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/docs) for this feature
- [ ] Yes, I ran `pre-commit run -a` with this PR

**Note**: Not all the PRs require a new example and/or doc page. In general:
- Use an existing example when possible to demonstrate a new addons usage
- A new docs page under `docs/add-ons/*` is required for new a new addon

### For Moderators

- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

**Comment by FernandoMiguel** [comment]
great stuff here.
thanks for putting it out

**Comment by Hokwang** [comment]
OMG, it wil be big change.

In 5.x structure, you miss "terraform-aws-eks-teams"

**Comment by bryantbiggs** [comment]
> In 5.x structure, you miss "terraform-aws-eks-teams"

Ah yes, thank you. Theres a few other details missing as well that I'll add soon

**Comment by bryantbiggs** [comment]
here is an early look at what v5 might look like https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/refactor/v5-part-deux

**Review by askulkarni2 — APPROVED** [review]

File: DIRECTION_v5.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+├── aws-quickstart/
+|   └── cdk-eks-blueprints/
+|       └── examples/
+├── aws-ia/
+|   ├── ecs-blueprints/
+|   |   └── examples/
+|   ├── terraform-aws-eks-blueprints/
+|   |   └── examples/
+|   ├── eks-addons/                      # Contains addons in supported formats (Terraform, ArgoCD, and FluxCD (future))
+|   ├── terraform-aws-eks-addon/         # Module for creating Terraform based addon (IRSA + Helm chart)
```
Comment (vara-bonthu) [inline DIRECTION_v5.md:97]: Is this `terraform-aws-kubernetes-addon` ?

File: DIRECTION_v5.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+|   ├── terraform-aws-eks-ack-addons/    # Will contain addon support for multiple frameworks in one location
+|   |   └── modules/
+|   |       ├── aws-load-balancer-controller/
+|   |       |   ├── main.tf
+|   |       |   ├── outputs.tf
+|   |       |   ├── variables.tf
+|   |       |   └── versions.tf
+|   |       └── <other-addons>/
+|   ├── terraform-aws-eks-multi-tenancy/ # Module for creating Kubernetes multi-tenancy constructs
```
Comment (vara-bonthu) [inline DIRECTION_v5.md:106]: Is this `terraform-aws-eks-multi-tenancy` same as `terraform-aws-eks-teams`?

File: DIRECTION_v5.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+2. The `aws-kms` module will be removed entirely. This was consumed in the root project module for cluster secret encryption. In its place, users can utilize the KMS key creation functionality of the [`terraform-aws-eks`](https://github.com/terraform-aws-modules/terraform-aws-eks) module or the [`terraform-aws-kms`](https://github.com/terraform-aws-modules/terraform-aws-kms) module if they wish to control the key separately from the cluster itself.
+3. The `emr-on-eks` module will be removed entirely; its replacement can be found in the addons under [`emr-on-eks`](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/modules/kubernetes-addons/emr-on-eks) and you can see an example of its usage under the [`emr-on-eks-fargate`](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples/analytics/emr-on-eks-fargate) example.
```
Comment (vara-bonthu) [inline DIRECTION_v5.md:37]: Let's update this will be replaced with new terraform module `terraform-emr-eks-containers`

**Review by vara-bonthu — COMMENTED** [review]
Few comments

File: DIRECTION_v5.md (hunk 2 above)
Comment (bryantbiggs) [reply DIRECTION_v5.md]: name updated in https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/1319/commits/0a8023866746c8be15c587b283c4090c8254d63d

File: DIRECTION_v5.md (hunk 1 above)
Comment (bryantbiggs) [reply DIRECTION_v5.md]: name updated in https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/1319/commits/0a8023866746c8be15c587b283c4090c8254d63d

**Comment by bryantbiggs** [comment]
shh - bad bot

**Pull request merged**